src/MT.lua

```diff
@@ -1174,7 +1175,13 @@ function M.getMTfromFile(self,tt)
    activeT = nil  -- done with activeT
    if (#aa > 0) then
       sort(aa)
-      LmodWarning{msg="w_Mods_Not_Loaded",module_list=concatTbl(aa," ")}
+      if (force ~= true) then
+      	      LmodWarning{msg="w_Missing_Coll", collectionName = collectionName, module_list = concatTbl(aa,"\", \"")}
+	      LmodErrorExit()
+	      return false
```

Comment:
this is a change from the current behaviour?

Reply:
Indeed. And I considered that as a bug, Robert too as it seems :)